src/app/core/common-components/entity-form/entity-form.service.ts

```diff
@@ -19,12 +19,27 @@ import { Subscription } from "rxjs";
 import { filter } from "rxjs/operators";
 import { EntitySchemaField } from "../../entity/schema/entity-schema-field";
 import { DefaultValueService } from "../../entity/default-value.service";
+import { DefaultValueConfig } from "../../entity/schema/default-value-config";
 
 /**
  * These are utility types that allow to define the type of `FormGroup` the way it is returned by `EntityFormService.create`
  */
-export type TypedForm<T> = FormGroup<{ [K in keyof T]: ɵElement<T[K], null> }>;
-export type EntityForm<T extends Entity> = TypedForm<Partial<T>>;
+export type TypedFormGroup<T> = FormGroup<{
+  [K in keyof T]: ɵElement<T[K], null>;
+}>;
+
+export type PartialTypedFormGroup<T extends Entity> = TypedFormGroup<
+  Partial<T>
+>;
+
+export interface ExtendedEntityForm<T extends Entity> {
```

Comment:
suggested naming:
- EntityFormGroup (instead of PartialTypedFormGroup)
- EntityForm (instead of ExtendedEntityForm - it has extended details but we don't have a simple/unextended one to contrast and shorter is better :wink: )
- accordingly  createExtendedEntityForm() -->  createEntityForm

Reply:
Yes, this was just the naming for the refactoring.

Reply:
I renamed some of the suggestions already, feel free to rename the other ones